### DISCUSSION TITLE
Disable JWT

### DISCUSSION THREAD

**Discussion by Enotukish** [ONLYOFFICE/Docker-DocumentServer#544] [Q&A] [answered]
Good evening, I ran into a problem connecting onlyoffice to nextcloud. I assume that the problem is in JWT, how to disable it?

**Comment by ShockwaveNN** [comment, marked as answer]
Hi, set [JWT_ENABLED](https://github.com/ONLYOFFICE/Docker-DocumentServer#available-configuration-parameters) parameter to false

But please be careful if this service is public, disabling JWT have several CVE with it
